roles/openshift_aws/templates/user_data.j2

```diff
@@ -2,23 +2,35 @@
 {{ openshift_aws_node_user_data }}
 {% else %}
 #cloud-config
+
 write_files:
 - path: /root/openshift_bootstrap/openshift_settings.yaml
   owner: 'root:root'
   permissions: '0640'
   content: |
-    openshift_node_config_name: {{ openshift_aws_node_group.node_group_config | default('unset') }}
-{%   if openshift_aws_node_group.group != 'master' %}
+    openshift_node_config_name: {%
```

Comment:
I don't think there's a need to modify this file at all.  Just don't pass this user data to the masters.

Reply:
user_data must be passed to masters and executed by cloud-init.  see runcmd: ansible-playbook, /root/openshift_bootstrap/bootstrap.yml.

Reply:
Fair enough.